recipes/mysql-connector-cpp/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,7 @@
+#include <mysqlx/xdevapi.h>
+
+int main()
+{
+    mysqlx::SessionSettings from_url("mysqlx://user:pwd@127.0.0.1:1234/db?ssl-mode=required");
```

Comment:
Hi!
Is this constructor accessing the network layer? 
In that case, new policies in Conan enforce not to access network layer from test_packages for security reasons so another library test could be done.

Reply:
It deliberately constructs a `SessionSettings` object instead of a `Session` one to avoid the creation of a network connection.

Reply:
Okey, thanks for the clarification!
It may be interesting to put a comment above to clarify this "non network layer access behavior" for future reviews!